mantle/coreir/logic.py

```diff
@@ -319,6 +319,27 @@ def DefineInvert(width):
 def Invert(width=None, **kwargs):
     return DefineInvert(width)(**kwargs)
 
+@cache_definition
+def DefineWire(width):
+    T = Bits(width)
+    decl = DeclareCircuit("Wire{}".format(width),
+            'in', In(T), 'out', Out(T),
+            simulate       = simulate_bits_invert,
```

Comment:
Looks like this needs to be updated, the simulate function should be trivial (just pass through the value), I can puh the change.

Reply:
Oops missed this during the copy and paste of the invert...